Makefile

```diff
@@ -66,6 +66,13 @@ DOCKER_RM_CMD=docker rm -f
 WORKSPACE_MAPPING_DIR=/workspace
 BUILD_CONTAINER_NAME=payload_sdk_build_env
 
+# To help in building sample cpp application which is located inside examples/app-cpp.
+# NOTE: If your app location is different then you need to provide the path here, and this path must exist within SatOS-Payload-SDK only. After changing go ahead and change SRC_DIR path.
```

Comment:
please add this elaborate comment in README than here. Here the comment can be 'application source folder'. Should be within SatOS-Payload-SDK directory.

Reply:
OK

Reply:
Done